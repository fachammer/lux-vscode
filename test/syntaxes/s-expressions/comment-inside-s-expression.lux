(function ##comment no arg)
(function #(block comment) arg)