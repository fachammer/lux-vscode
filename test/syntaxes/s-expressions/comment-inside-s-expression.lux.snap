>(function ##comment no arg)
#^ source.lux meta.expression.lux punctuation.section.expression.begin.lux
# ^^^^^^^^ source.lux meta.expression.lux entity.name.function.lux
#         ^ source.lux meta.expression.lux
#          ^^ source.lux meta.expression.lux comment.line.double-hash.lux
#            ^^^^^^^^^^^^^^^ source.lux meta.expression.lux comment.line.double-hash.lux
>(function #(block comment) arg)
#^ source.lux meta.expression.lux meta.expression.lux punctuation.section.expression.begin.lux
# ^^^^^^^^ source.lux meta.expression.lux meta.expression.lux entity.name.function.lux
#         ^ source.lux meta.expression.lux meta.expression.lux
#          ^^ source.lux meta.expression.lux meta.expression.lux comment.block.lux
#            ^^^^^^^^^^^^^ source.lux meta.expression.lux meta.expression.lux comment.block.lux
#                         ^ source.lux meta.expression.lux meta.expression.lux comment.block.lux
#                          ^ source.lux meta.expression.lux meta.expression.lux
#                           ^^^ source.lux meta.expression.lux meta.expression.lux meta.symbol.clojure
#                              ^ source.lux meta.expression.lux meta.expression.lux punctuation.section.expression.end.trailing.lux